synthetic_data/synthetic_data.py

```diff
@@ -305,9 +306,112 @@ def make_tabular_data(
     #
 
     if noise_level_x > 0.0:
-        x_noise = generate_x_noise(
-            x_final[:, :n_informative], noise_level_x, seed=seed
-        )
+        x_noise = generate_x_noise(x_final[:, :n_informative], noise_level_x, seed=seed)
         x_final[:, :n_informative] = x_final[:, :n_informative] + x_noise
 
     return x_final, y_reg, y_prob, y_labels
+
+
+def make_data_from_report(
```

Comment:
```
profile = Profiler(data)
readable_report = profile.report(report_options={"output_format": "compact"})
print(json.dumps(readable_report, indent=4))
```

I think it is named `_report` because what is being passed to the function is actually what we refer to as a `report` --> the readable dictionary returned from `profile.report()`